src/main.rs

```diff
@@ -485,6 +485,7 @@ mod unit_tests {
                     file: None,
                     mergeable: false,
                     hotplug_size: None,
+                    virtiomem_size: None,
```

Comment:
Reuse `hotplug_size` and add an enum `HotplugMethod` with two types `ACPI`, `VirtioMem`

Reply:
Done.